tests/testthat/_snaps/geodist.md

```diff
@@ -96,7 +96,8 @@
     Condition
       Error in `step_geodist()`:
       Caused by error in `prep()`:
-      ! `lat` should resolve to a single column name.
+      x The `lat` selector should select at most a single variable.
+      i The following 2 were selected:`x` and `x1`.
```

Comment:
```suggestion
      i The following 2 were selected: `x` and `x1`.
```

Reply:
done in https://github.com/tidymodels/recipes/pull/1255/commits/483af8a84507657d909bec152f8bb87e15fb9350